src/pokedex_area_screen.c

```diff
@@ -806,3 +811,9 @@ static void CreateAreaUnknownSprites(void)
         }
     }
 }
+
+static void LoadHGSSScreenSelectBarSubmenu(void)
+{
+    CopyToBgTilemapBuffer(1, sPokedexPlusHGSS_ScreenSelectBarSubmenu_Tilemap, 0, 0);
```

Comment:
I would return early if `POKEDEX_PLUS_HGSS` is false to avoid building it into the rom if disabled

Reply:
Added in a check just like the one in CB2_OpenPokedexPlusHGSS().

Reply:
Hm, I think there is no need to return it early. because it is taken care of above
```
        if (POKEDEX_PLUS_HGSS)
            LoadHGSSScreenSelectBarSubmenu();
```

Reply:
Check if by compiling with the dex off it includes any test exclusive to the HGSS Dex by using HMA

Reply:
Per Edu's suggestion, I dug through the ROM in HMA and didn't find HGSS dex specific text present when the dex was turned off and there was no check to exit early in LoadHGSSScreenSelectBarSubmenu(). So I've removed it.

Reply:
Apologies, my bad then! Sorry for the unnecessary extra work :)